src/lv_draw/lv_draw_img.c

```diff
@@ -425,6 +425,41 @@ lv_img_src_t lv_img_src_get_type(const void * src)
     return img_src_type;
 }
 
+lv_img_dsc_t *lv_img_buf_alloc(lv_coord_t w, lv_coord_t h, lv_img_cf_t cf)
+{
+    /*Get the pixel size in bytes*/
+    uint8_t px_b = lv_img_color_format_get_px_size(cf) / 8;
```

Comment:
px_b can be easily 0 with e.g. LV_IMG_CF_INDEXED_2BIT where the functio nreturn 2.